tests/fibgo/connection_test.go

```diff
@@ -318,7 +396,12 @@ func TestFib(t *testing.T) {
 			t.Errorf("GenerateFib error %v", err)
 			return
 		}
-		_, err = waitForReceipt(client, tx, common.NewAddressFromEth(session.TransactOpts.From), time.Second*60)
+		_, err = waitForReceipt(
+			client,
+			tx,
+			common.NewAddressFromEth(session.TransactOpts.From),
+			time.Second*60,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '60', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)